tools/cibsecret.in

```diff
@@ -14,6 +14,23 @@
 # Secrets are ASCII files, holding one value per file:
 # <secrets-directory>/<rsc>/<param>
 
+# These constants must track crm_exit_t values
+CRM_EX_OK=0
+CRM_EX_ERROR=1
+CRM_EX_INVALID_PARAM=2
+CRM_EX_UNIMPLEMENT_FEATURE=3
+CRM_EX_INSUFFICIENT_PRIV=4
+CRM_EX_USAGE=64
+CRM_EX_CONFIG=78
+CRM_EX_OLD=103
+CRM_EX_DIGEST=104
+CRM_EX_NOSUCH=105
+CRM_EX_UNSAFE=107
+CRM_EX_EXISTS=108
+CRM_EX_MULTIPLE=109
+CRM_EX_EXPIRED=110
+CRM_EX_NOT_YET_IN_EFFECT=111
```

Comment:
It's better to list only the values the script actually uses, because shellcheck will complain about unused variables.